File: Java/Java source-code/get_array_value.java


public class get_array_value {
    public static int get_array_value(Integer[] a, Integer k) {
        if (k - 1 >= a.length || k - 1 < 0) {
            return -100000;
        } else {
            return a[k - 1];
        }
    }
}

File: Methods/Methods-C++/get_array_value.cpp
#include <iostream>
#include <math.h>
#include <algorithm>

int get_array_value(int a[], int k, int n) {
    if(k - 1 >= n || k - 1 < 0) {
        return -100000;
    } else {
        return a[k - 1];
    }
}